packages/vite-plugin-cloudflare/src/__tests__/get-warning-for-workers-resolved-configs.spec.ts

```diff
@@ -0,0 +1,179 @@
+						nonApplicable: {
+							replacedByVite: new Set([]),
+							notRelevant: new Set([]),
+							overridden: new Set(),
+						},
+						raw: {} as any,
+					},
+					{
+						type: 'worker',
+						config: {
+							configPath: './b/wrangler.json',
+						} as Partial<WorkerConfig> as WorkerConfig,
+						nonApplicable: {
+							replacedByVite: new Set([]),
+							notRelevant: new Set([]),
+							overridden: new Set(),
+						},
+						raw: {} as any,
+					},
+				],
+			});
+			expect(warning).toBeUndefined();
+		});
+	});
+
+	test('entry worker only', () => {
+		const warning = getWarningForWorkersConfigs({
+			entryWorker: {
+				type: 'worker',
+				config: {
+					name: 'entry-worker',
+					configPath: './wrangler.json',
+				} as Partial<WorkerConfig> as WorkerConfig,
+				nonApplicable: {
+					replacedByVite: new Set(['alias', 'minify']),
+					notRelevant: new Set([
+						'build',
+						'find_additional_modules',
+						'no_bundle',
+					]),
+					overridden: new Set(['rules']),
+				},
+				raw: {} as any,
+			},
+
+			auxiliaryWorkers: [],
+		});
+		expect(warning).toMatchInlineSnapshot(`
+			"
+
+			[43mWARNING[0m: your worker config (at \`wrangler.json\`) contains the following configuration options which are ignored since they are not applicable when using Vite:
+			  - \`alias\` which is replaced by Vite's \`resolve.alias\` (docs: https://vite.dev/config/shared-options.html#resolve-alias)
+			  - \`minify\` which is replaced by Vite's \`build.minify\` (docs: https://vite.dev/config/build-options.html#build-minify)
+			  - \`build\`, \`find_additional_modules\`, \`no_bundle\` which are not relevant in the context of a Vite project
+			  - \`rules\` which is overridden by \`@flarelabs-net/vite-plugin-cloudflare\`
+			"
+		`);
+	});
+
+	test('multi workers', () => {
+		const warning = getWarningForWorkersConfigs({
+			entryWorker: {
+				type: 'worker',
+				config: {
+					name: 'entry-worker',
+					configPath: './wrangler.json',
+				} as Partial<WorkerConfig> as WorkerConfig,
+				nonApplicable: {
+					replacedByVite: new Set(['alias']),
+					notRelevant: new Set(['build']),
+					overridden: new Set(),
+				},
+				raw: {} as any,
+			},
+			auxiliaryWorkers: [
+				{
+					type: 'worker',
+					config: {
+						name: 'worker-a',
+						configPath: './a/wrangler.json',
+					} as Partial<WorkerConfig> as WorkerConfig,
+					nonApplicable: {
+						replacedByVite: new Set([]),
+						notRelevant: new Set(['find_additional_modules', 'no_bundle']),
+						overridden: new Set(),
+					},
+					raw: {} as any,
+				},
+				{
+					type: 'worker',
+					config: {
+						configPath: './b/wrangler.json',
+					} as Partial<WorkerConfig> as WorkerConfig,
+					nonApplicable: {
+						replacedByVite: new Set([]),
+						notRelevant: new Set(['site']),
+						overridden: new Set(),
+					},
+					raw: {} as any,
+				},
+			],
+		});
+		if (process.platform === 'win32') {
```

Comment:
Generally what we do with windows and snapshots is to post process the warning text to convert any windows paths to unix ones. But I guess this is also OK, although it means that to update the snapshots you must either run locally on Windows with `-u` (which is a pan) or manually change the snapshot (which is not recommended).

Reply:
ah ok, yours sounds like a better approach! I'll try that 🙂👍 

(yeah here I've been manually updating the snapshot 😓)

Reply:
updated 🙂